src/main/java/ch/wisv/areafiftylan/web/controller/WebRestController.java

```diff
@@ -21,13 +20,15 @@
     TournamentServiceImpl tournamentService;
     EventServiceImpl eventService;
     SponsorServiceImpl sponsorService;
+    CommitteeMemberServiceImpl committeeService;
```

Comment:
This is not how you're supposed to use Services in the controller. Services have an interface, and an implementation and you declare the interface as a field in your controller. A service for each web entity might be a little overkill, zo maybe combine these into two or three services (Something like EventService for tournaments, events and other... events) and a Webservice for other static content such as sponsors, committees and such.